example/ida/utils.py

```diff
@@ -132,7 +132,7 @@ def __init__(self, expr):
         self.expr = expr
 
         # Initial translation
-        text = Translator.to_language(self.languages[0]).from_expr(self.expr)
+        text = Translator.to_language(self.languages[0])().from_expr(self.expr)
```

Comment:
Yep, I agree with you. The use of the instance seems to be the 99% case, not the class use.

I'll update the PR ASAP accordingly (and thus keep compatibility with the old API).